azure/Guidelines.md

```diff
@@ -18,6 +20,35 @@ Teams building ARM Resource Providers (RPs) MUST follow the additional guidance
 
 ARM RPs are Azure Fundamentals requirement for Azure Services and ARM RP review is another mandatory review. Some of the guidance overlaps with general API review, but passing one review will generally make the other one go very quickly.
 
+### Advice for new services
+
+Developing a new service requires the development of at least 1 (management plane) API and potentially one or more additional (data plane) APIs.  
+
+* Think about naming from the context of a **developer experience**.  
```

Comment:
[RyanSweet] Think about the code that the customer will write (in at least two languages - a dynamic language like Python or JavaScript, and a static language like Java or C#) both before and after the REST API call.  How will a developer use this API in the standard use case?